biogeophys/FatesPlantRespPhotosynthMod.F90

```diff
@@ -361,8 +364,11 @@ real(r8) :: tcwood             ! Temperature response function for wood
                      ft = currentCohort%pft
                      cl = currentCohort%canopy_layer
                      
-                     call bleaf(currentCohort%dbh,currentCohort%pft,currentCohort%canopy_trim,b_leaf)
-                     call storage_fraction_of_target(b_leaf, &
+                     call bleaf(currentCohort%dbh,currentCohort%pft,currentCohort%canopy_trim,store_c_target)
+!                     call bstore_allom(currentCohort%dbh,currentCohort%pft, &
```

Comment:
@ckoven after revisiting this, why are we passing the leaf target carbon into the "storage_fraction_of_target" routine here?  I re-labeled the argument into the routine as store_c_target, but maybe that is not what you intended.  Because the store_c_target would actually have the cushion multiplier incorporated into it (which would be used if we called bstore_allom()).  Here is a link to storage_fraction_of_target() for quick reference:

https://github.com/NGEET/fates/blob/master/biogeochem/FatesAllometryMod.F90